src/convert-workspace.ts

```diff
@@ -106,6 +106,13 @@ export default async function convert(options: WorkspaceConversionSettings):
   // Process & write each conversion result:
   await writeFileResults(options.workspaceDir, converter.getResults());
 
+  // Delete files that were explicitly requested to be deleted.
+  for (const repo of options.reposToConvert) {
+    for (const glob of options.deleteFiles || []) {
+      await rimraf(path.join(repo.dir, glob));
```

Comment:
So this means that the globs apply to all repos, and we can't write a glob that just applies to one by prefixing the repo name? Fine either way, but clarify in a comment.

Reply:
Done.